samples/spring-valueentity-customer-registry/src/main/java/com/example/api/CustomersResponseByName.java

```diff
@@ -0,0 +1,23 @@
+package com.example.api;
+
+import com.example.domain.Customer;
+import kalix.javasdk.view.View;
+import kalix.springsdk.annotations.Query;
+import kalix.springsdk.annotations.Subscribe;
+import kalix.springsdk.annotations.Table;
+import kalix.springsdk.annotations.ViewId;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@ViewId("view_response_customers_by_name")
+@Table("customers_by_name")
+// tag::class[]
+@Subscribe.ValueEntity(CustomerEntity.class)
+public class CustomersResponseByName extends View<Customer> { // <1>
```

Comment:
moved to views package for consistency. tks

Reply:
Yup, it's very hard to draw the line. I think I will leave it within`views` for consistency but it will be good to get some common understanding of how to approach that and if we want to change all the samples to be consistent with whatever we decide.

To be honest, at times I think it can be harder if you have 6, 7, 8 files all under the same package rather than having them separated. So I'm not sure I agree with the extra brain cycle usage but that's one for later.

Reply:
Being opinionated is not always a bad thing, because I find it quite useful if someone gives me a direction about something, in this case, packaging. If I prefer to go with a different packing I will do so. And personally, I think that it requires more brain cycles to filter classes that are not relevant for understanding a given concept (like @efgpinto said). 

What is an API and what's not is a separate discussion. Agree with that.